libs/utils/utils.js

```diff
@@ -46,33 +46,43 @@ const AUTO_BLOCKS = [
   { 'pdf-viewer': '.pdf' },
 ];
 const ENVS = {
-  local: { name: 'local' },
+  local: {
+    name: 'local',
+    edgeConfigId: '8d2805dd-85bf-4748-82eb-f99fdad117a6',
+  },
   stage: {
     name: 'stage',
     ims: 'stg1',
     adobeIO: 'cc-collab-stage.adobe.io',
     adminconsole: 'stage.adminconsole.adobe.com',
     account: 'stage.account.adobe.com',
+    edgeConfigId: '8d2805dd-85bf-4748-82eb-f99fdad117a6',
   },
   prod: {
     name: 'prod',
     ims: 'prod',
     adobeIO: 'cc-collab.adobe.io',
     adminconsole: 'adminconsole.adobe.com',
     account: 'account.adobe.com',
+    edgeConfigId: '2cba807b-7430-41ae-9aac-db2b0da742d5',
   },
 };
 
-function getEnv() {
+function getEnv(conf) {
   const { host, href } = window.location;
   const location = new URL(href);
   const query = location.searchParams.get('env');
 
-  if (query) { return ENVS[query]; }
-  if (host.includes('localhost:')) return ENVS.local;
+  if (query) return { ...ENVS[query], consumer: conf[query] };
+  if (host.includes('localhost:')) return { ...ENVS.local, consumer: conf.local };
```

Comment:
The current milo `config` obj does not have local/stage/prod values. 
1. Should we update ours to demonstrate how it can be used?
2. Is it expected that values that are shared across all envs would be duplicated in the config object?

I may just be confused though, as the `conf` obj provided by consumers follows the `ENVS` format, but in Milo we pass in the `config` obj in scripts.js (line 25) which does not differentiate between envs.

Reply:
This is a new _non required_ set of children in the config object. BACOM has been updated since they need it for this feature.